src/pages/home/report/AnimatedEmptyStateBackground.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Animated, {SensorType, useAnimatedSensor, useAnimatedStyle, useSharedValue, withSpring} from 'react-native-reanimated';
+import useWindowDimensions from '../../../hooks/useWindowDimensions';
+import * as NumberUtils from '../../../libs/NumberUtils';
+import EmptyStateBackgroundImage from '../../../../assets/images/empty-state_background-fade.png';
+import * as StyleUtils from '../../../styles/StyleUtils';
+
+const propTypes = {
+    /** Is the window width narrow, like on a mobile device */
+    isSmallScreenWidth: PropTypes.bool.isRequired,
```

Comment:
This prop is not needed since you're using `useWindowDimensions` already. Let's remove it.